training/level-2-command-line-interfaces/dragon-warrior/tmarsha1/file_ops.py

```diff
@@ -1,21 +1,125 @@
-"""This module provides various functions for operating on files"""
+"""
+This module provides various functions for operating on files
+
+Functions:
+    copy_files: copy file(s) to a specific location.
+    move_files: move file(s) to a specific location.
+"""
 
 import subprocess
 
 
-def copy_files(files: list, destination: str):
+class FolderNotFoundError(Exception):
```

Comment:
@TroyLMarshall Excellent.  Custom exceptions.  It is common to see exceptions defined in a separate module.  Something to consider.